runtime/lua/vim/health.lua

```diff
@@ -4,22 +4,140 @@ function M.report_start(msg)
   vim.fn['health#report_start'](msg)
 end
 
+M.start = M.report_start
+
 function M.report_info(msg)
   vim.fn['health#report_info'](msg)
 end
 
+M.info = M.report_info
+
 function M.report_ok(msg)
   vim.fn['health#report_ok'](msg)
 end
 
+M.ok = M.report_ok
+
 function M.report_warn(msg, ...)
   vim.fn['health#report_warn'](msg, ...)
 end
 
+M.warn = M.report_warn
+
 function M.report_error(msg, ...)
   vim.fn['health#report_error'](msg, ...)
 end
 
+M.error = M.report_error
+
+function M.provider_disabled(provider)
```

Comment:
User will be able to access this function. Should we perhaps add an underscore prefix to mark it's for internal use only?